src/metadata.rs

```diff
@@ -129,6 +153,34 @@ impl Metadata {
     pub fn extension(&self) -> Option<&str> {
         self.extension.as_deref()
     }
+    /// Returns true the number of episodes is greater than one.
+    ///
+    ///```
+    /// # use torrent_name_parser::Metadata;
+    ///
+    /// if let Ok(m) = "Life.on.Mars.(UK).S01E1.avi".parse::<Metadata>() {
+    ///    assert_eq!(m.is_multi_episode(), false);
+    ///    assert_eq!(m.episode(), Some(1));
+    /// }
+    ///
+    /// if let Ok(m) = "Life.on.Mars.(UK).S01E1E02.avi".parse::<Metadata>() {
+    ///    assert_eq!(m.is_multi_episode(), true);
+    ///    assert_eq!(m.episode(), Some(1));
+    ///    assert_eq!(m.last_episode(), Some(2));
+    /// }
+    ///
+    /// if let Ok(m) = "Life.on.Mars.(UK).S01E1E02E03.avi".parse::<Metadata>() {
+    ///    assert_eq!(m.is_multi_episode(), true);
+    ///    assert_eq!(m.episode(), Some(1));
+    ///    assert_eq!(m.last_episode(), Some(3));
+    /// }
+    ///```
+    pub fn is_multi_episode(&self) -> bool {
+        self.last_episode.is_some()
+    }
+    pub fn multi_episode(&self) -> Option<&Vec<i32>> {
```

Comment:
Pro tip: never return references to `Vec`, always return references to slices as `&Vec<i32>` implements `&[i32]` and will convert automatically.

Reply:
I get the pro tip. So far I have not been able to avoid the `&Vec`

Reply:
I think perhaps. I just don't need the `Option<>`. Vecs can be empty.

Reply:
You might be missing a `.transpose()` or something to bubble up the reference.

But you're right, we can do away with the `Òption` here and just use an empty vec on a `None`.